platform-includes/getting-started-config/javascript.capacitor.mdx

```diff
@@ -3,7 +3,7 @@ Then forward the `init` method from the sibling Sentry SDK for the framework you
 
 ```typescript {tabTitle: Angular} {filename: app.module.ts} {"onboardingOptions": {"performance": "13-16, 21-28, 46-55", "session-replay": "17-19, 29-33"}}
```

Comment:
would be great to have both snippets in separate tabs, wdyt?

Reply:
That is a good idea! Since it was a small change, I decided to also add the snippet for React and Vue